web/main.js

```diff
@@ -19,15 +19,14 @@ let signalserver = new URL(location.href);
 // peerconnection is the active connection's WebRTC object. Global to help debugging.
 let peerconnection;
 // UI elements.
-let filepicker;
 let dialButton;
 let phraseInput;
-let clipboardInput;
 let mainForm;
 let qrImg;
```

Comment:
Following [this comment](https://github.com/SpiderOak/webwormhole/pull/6#discussion_r749606499), we can remove this as well, right?

Reply:
Yes we can.